front/app/services/projectFolderImages.ts

```diff
@@ -1,21 +1,42 @@
-import { ImageSizes } from 'typings';
 import { API_PATH } from 'containers/App/constants';
 import streams, { IStreamParams } from 'utils/streams';
+import { TProjectFolderCardSize } from 'components/ProjectAndFolderCards/components/ProjectFolderCard';
+import { CARD_IMAGE_ASPECT_RATIO as PROJECT_CARD_IMAGE_ASPECT_RATIO } from 'services/projectImages';
 
 const apiEndpoint = `${API_PATH}/project_folders`;
 
+export const CARD_IMAGE_ASPECT_RATIO = PROJECT_CARD_IMAGE_ASPECT_RATIO;
+
+export type ProjectFolderImageSizes = {
+  small: string | null;
+  large: string | null;
+};
+
+export const getCardImageUrl = (
+  imageVersions: ProjectFolderImageSizes,
+  isPhone: boolean,
+  cardSize?: TProjectFolderCardSize
+) => {
+  if (isPhone || cardSize !== 'small') {
```

Comment:
I've just realized that if `cardSize` is not passed, this condition will always be `true`.

Reply:
But maybe it's fine.

Reply:
Double checking it. I'm also seeing if we can make it clearer when we'll have which cardSize.

Reply:
I indeed think it's fine. The most used cardSize when isPhone is false will be `medium` or `large`, so this condition would be true most often anyway. 

I've based this on: if the card is on the first page (first 6 cards) + in a dynamic layout, it'll be medium or large. A dynamic layout is what we use on landing pages (home + custom page).

To get this completely right, we'd also need a full preview of the different project cards, rather than device. But that would be a bit more work.